cvmfs/swissknife_sync.h

```diff
@@ -12,6 +12,13 @@
 #include "upload.h"
 
 struct SyncParameters {
+  static const unsigned DEFAULT_MAX_WEIGHT = 100000;
```

Comment:
Please change to match the naming convention for constants, e.g. `kDefaultMaxWeight`